Gruntfile.js

```diff
@@ -72,16 +72,6 @@ module.exports = function (grunt) {
 				maxBuffer: maxBufferSize
 			},
 
-			coverageUnit: {
```

Comment:
We should relying just on `coverageSingle` task. Developers should test only the file they are using locally. I'll remove those `index.js` within the next commit.